app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAgeShouldReturnExpectedResult:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
+            (-1, -1, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ]
+    )
+    def test_get_human_age_returns_correct_age_for_cat_and_dog(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected_result: list
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == expected_result
```

Comment:
Add test cases to test against invalid inputs and raised errors, when string, dict, list, etc. is passed to the function. This test is mentioned in the checklist as well

Reply:
added test